src/linagora.esn.contact/app/app.routes.js

```diff
@@ -69,14 +69,14 @@ require('./services/contact-configuration.service.js');
         })
         .state('contact.addressbooks.new', {
           url: '^/contact/new/:bookId/:bookName',
-          views: {
-            'main@contact': {
-              template: require('./contact/create/contact-create.pug'),
-              controller: 'newContactController',
-              resolve: {
+          resolve: {
+            modalInstance: function($modal) {
+              return $modal({
+                template: require('./contact/create/contact-create.pug'),
+                controller: 'newContactController',
                 domain: routeResolver.session('domain'),
                 user: routeResolver.session('user')
-              }
+              });
```

Comment:
you can use the following code to hide the modal when leaving this state
```
          onExit: function(modalInstance) {
            modalInstance.hide();
          }
```

Reply:
the some comment ?